src/pykeen/nn/emb.py

```diff
@@ -591,14 +591,15 @@ def __init__(
         embedding_specification: EmbeddingSpecification,
         num_layers: int = 2,
         use_bias: bool = True,
-        use_batch_norm: bool = False,
         activation: Hint[nn.Module] = None,
         activation_kwargs: Optional[Mapping[str, Any]] = None,
         edge_dropout: float = 0.4,
         self_loop_dropout: float = 0.2,
         edge_weighting: Hint[EdgeWeighting] = None,
         decomposition: Hint[Decomposition] = None,
         decomposition_kwargs: Optional[Mapping[str, Any]] = None,
+        regularizer: Hint[Regularizer] = None,
+        regularizer_kwargs: Optional[Mapping[str, Any]] = None,
     ):
         """Instantiate the R-GCN encoder.
```

Comment:
is there a dangling doc for `use_batch_norm` now?

Reply:
b37f024